src/libs/Localize/index.ts

```diff
@@ -162,6 +127,13 @@ function getTranslatedPhrase<TKey extends TranslationPaths>(
     return getTranslatedPhrase(CONST.LOCALES.DEFAULT, phraseKey, null, ...parameters);
 }
 
+const memoizedGetTranslatedPhrase = memoize(getTranslatedPhrase, {
+    maxArgs: 2,
+    equality: 'shallow',
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
+    skipCache: ([language, phraseKey, fallbackLanguage, ...parameters]) => !isEmptyObject(parameters.at(0)),
```

Comment:
If we access only fourth parameter, would that be possible to avoid rest operator and disabling eslint rule with:

```suggestion
    skipCache: (params) => !isEmptyObject(params.at(3)),
```

Reply:
@shubham1206agra can you please address?